### ISSUE TITLE
How to train the deeplab v3?

### ISSUE THREAD

**Issue by MrLinNing** [speedinghzl/pytorch-segmentation-toolbox#13]
Can you provide more details about training deeplab v3 on cityscapes dataset?
Besides the mIoU=78.30 about the PSPnet is the valid result, and how about the test results of cityscapes dataset?  Can you give the detail of training it? Thank you! @speedinghzl

**Comment by speedinghzl** [comment]
@MrLinNing Our reproduced PSPNet can achieve ~78.5 on val set and ~77.5 on test set using single scale testing.
You can run Deeplabv3 by replacing this [line](https://github.com/speedinghzl/pytorch-segmentation-toolbox/blob/master/train.py#L16) and keeping the hyperparameters the same.
After the conference deadline, I will reorganize the code and support more models and datasets. Thanks.
